### PR TITLE
New ebuild for minixml 2.9

### PR DESCRIPTION
**Pull request by chatcannon**
Small library, required for NeXus http://www.nexusformat.org/

### CODE REVIEW

File: dev-libs/minixml/minixml-2.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit eutils
```
Comment (jlec) [inline dev-libs/minixml/minixml-2.9.ebuild]: not used.

File: dev-libs/minixml/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<herd>sci</herd>
+<maintainer>
+  <email>cjk34@cam.ac.uk</email>
+  <name>Chris Kerr</name>
+  <description>I don't develop this software, I'm just a Gentoo user who wanted to install it</description>
+</maintainer>
+<longdescription>Mini-XML is a lightweight, ultra-portable XML library that requires only an ANSI C compiler and a 'make' program.</longdescription>
```
Comment (jlec) [inline dev-libs/minixml/metadata.xml]: longline.

**Comment by junghans** [comment]
Why don;t you call the package `mxml` ?

**Comment by chatcannon** [comment]
mxml is an abbreviation, the name of the software is "Mini-XML". I suppose I could have added a hyphen to the package name.

**Comment by junghans** [comment]
The reason why I was asking, is that the tarball name, and directory in there, is `mxml` and packages are usually named after that.

Also could you rebase/merge this change with latest master.

**Comment by junghans** [comment]
Debian calls it `mxml`, too. So I would prefer that package name.

**Comment by chatcannon** [comment]
Just submitted a new PR (#350) with the ebuild renamed to mxml

**Pull request closed without merging**